client/lib/query-manager/media/test/index.js

```diff
@@ -228,7 +228,7 @@ describe( 'MediaQueryManager', () => {
 			test( 'should return false if query is not ISO 8601', () => {
 				const isMatch = MediaQueryManager.matches(
 					{
-						before: '2018',
+						before: '18',
```

Comment:
Previous value `2018` is a valid ISO 8601 date (https://en.wikipedia.org/wiki/ISO_8601#Years) and the new version of `moment` supports it (https://github.com/moment/moment/pull/4470).

Reply:
Changing the behaviour of a function for an input is a breaking change, even if it's enabling new values. That can easily break an application. They should have made this a major version bump 😬